### ISSUE TITLE
Update vom 1.1.1 to 1.2.1 and includes not found

### ISSUE THREAD

**Issue by petershaw** [niftylettuce/node-email-templates#123]
Hi, 
I updated vom 1.1.1 to 1.2.1. My template folder looks like this

```
_footer
   html.ejs
_header
   html.ejs
   style.scss
   text.ejs
one
   html.ejs
   style.scss
   text.ejs
```

in one/style.scss is a directive:

```
@import "_header/style";
```

in 1.1.1 this works like expected. But in 1.2.1 _header/style can't be found. 

```
 error: file to import not found or unreadable: \\\"_header/style\\\"\\n\""
```

**Comment by niftylettuce** [comment]
@petershaw have you resolved?

**Comment by petershaw** [comment]
no. Sadly resolved by downgrade to 1.1.1.

**Comment by niftylettuce** [comment]
Don't you need to do "../_header" or something to fix the relative path?  Is this a bug with vom?
